src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/custom.py

```diff
@@ -605,15 +605,9 @@ def _generate_ssh_keys():
     """Generates ssh keys pair"""
     private_key_path = os.path.join(os.path.expanduser('~'), '.ssh', 'id_rsa')
     public_key_path = os.path.join(os.path.expanduser('~'), '.ssh', 'id_rsa.pub')
-    if os.path.exists(private_key_path) and os.path.exists(public_key_path):
-        logger.warning('Reusing existing ssh public key from ~/.ssh')
-        return
-    if os.path.exists(private_key_path):
-        logger.warning('SSH private key id_rsa exists but public key is missing. Please export the public key.')
-        return
     keys.generate_ssh_keys(private_key_path, public_key_path)
-    logger.warning('SSH key files id_rsa and id_rsa.pub have been generated under ~/.ssh to allow SSH access to the '
-                   'nodes. If using machines without permanent storage, back up your keys to a safe location.')
+    logger.warning('Attempted to find or generate SSH key files id_rsa and id_rsa.pub under ~/.ssh to allow SSH access '
```

Comment:
I am not so sure we need this warning any more, as azure-cli-core already emits out a warning based the context, right?

Reply:
Yes it does in most cases. However, with this change no message will be printed if a new private key file is generated. In the past there was always a message printed. Still, this should be expected as the help text for `--generate-ssh-keys` in `az batchai cluster/file-server create -h`  says: "Generate SSH public and private key files in ~/.ssh directory (if missing)."

Reply:
I will go ahead and remove this warning.